lms/djangoapps/courseware/models.py

```diff
@@ -136,6 +142,8 @@ class XBlockFieldBase(models.Model):
     """
     Base class for all XBlock field storage.
     """
+    objects = CallStackManager()
+
```

Comment:
Sorry, I meant to add the Mixin to `XBlockFieldBase`, so that you capture `save` and `delete` on those objects.